app/templates/components/chart-component.hbs

```diff
@@ -1,3 +1,3 @@
-<svg width={{outerWidth}} height={{outerHeight}}>
-  <g class="chart-viewport" transform={{transformViewport}}></g>
-</svg>
\ No newline at end of file
+<svg {{bind-attr width=outerWidth}} {{bind-attr height=outerHeight}}>
```

Comment:
this should not be pushed to master, I believe this was caused by the rebase confusion